Assignment2/sebufer-SergioBujanFernandez.json

```diff
@@ -0,0 +1,23 @@
+"@context":{
+	"rdfs":"http:www.oeg-upm.net/ontologies/class#",
+	"includes": "rdf:includes",
+	"hasMeasurament":  "rdf:hasMeasurament",
+	"hasTemperatura":  "rdf:integer",
+	"atTime":  "rdf:dataTime",
+	"hasOwner": "rdf:hasOwner",
+	"hasName":  "rdf:string"
+ }
+ "@id":"http:www.oeg-upm.net/ontologies/class/class01"{
+ 		
+ 		"@id": "http:www.oeg-upm.net/ontologies/includes/Sensor029"{
+ 			"@id":"http:www.oeg-upm.net/ontologies/hasMeasurament/Measurament8401",
+ 			"@types":{ 	"@hasTemperature": "29",
+ 						"@atTime": "2010-06-12T12:00:12"}

```

Comment:
Wrong definition, "@types" tag doesn't exist. Also, you don't need defining as @type. In addition, don't use @  in attribute names. Use the attribute names according to those included in @context.
Check the same comments for the next definitions in this file. 
(Also, hasOwner property is missing)